### DISCUSSION TITLE
Problem installing LazyVim | could not resolve host github.com | plugins cloning failed

### DISCUSSION THREAD

**Discussion by sarodriguezva** [LazyVim/LazyVim#1382] [Q&A] [answered]
I am installing LazyVim for the first time but when I run the 'nvim' command I get this list of errors:
![image](https://github.com/LazyVim/LazyVim/assets/55607589/580025f3-b114-4fd7-af18-7ab3188d6473)

I've tried syncing and installing many times, but I get always the same. I've done some tests with my connection by cloning other repos and pinging some domains (google.com and github.com) and I've discovered that all of that goes well at WSL startup or if I wait a while after trying to install the LazyVim plugins.

Here is what I get if I ping at WSL startup:
![image](https://github.com/LazyVim/LazyVim/assets/55607589/7613abae-23f8-4754-897b-4cf13d0b0bd4)

Just after trying to install the LazyVim plugins and getting the errors in the first image, I get this:
![image](https://github.com/LazyVim/LazyVim/assets/55607589/e3f4b489-721e-48c5-b2a2-90e81ec7fc38)

This error remains some minutes and then I can ping correctly again, but if I try to run 'nvim' again the same errors appears. I don't know what to do.

**Comment by renews** [comment]
+1

**Comment by dpetka2001** [comment]
This seems to be a problem with WSL network connectivity. Doubt it has anything to do with Lazyvim. You should google for WSL connectivity issues. You could see for example this [issue](https://github.com/microsoft/WSL/issues/5437#issuecomment-647161596) and see if it works for you or keep googling some other solution if it doesn't work.

**Reply by morenohernan** [reply]
Hi, anyone managed to find a solution for the network issues in WSL with Lazyvim?  I have similar problem
![image](https://github.com/LazyVim/LazyVim/assets/700619/0884205c-2fd2-45fc-849a-78e782a778ba)

**Reply by sarodriguezva** [reply, marked as answer]
For me temporary works editing the /etc/resolv.conf file; edit it as sudo and change the nameserver to 1.1.1.1
The problem seems to be an issue with my network providor but by doing the above the problem is solved at least while in the current session. Hope it helps!

**Comment by 128f** [comment]
For any future googlers that concluded they definitely didn't have a DNS issue, I changed the concurrency setting to 4 in the hopes that something had decided I was spamming:

```lua
require("lazy").setup("lazy-plugins", {
[...]
  concurrency = 4,
```

I also blew out my nvim share directory (`rm -rf ~/.local/share/nvim`).


I think it was the share directory that solved it, but like any good scientist I always make sure to change multiple things at once before I try again.


I had a bizarre set of circumstances including migrating away from `nix` and picking my configs up and moving them from a mac so ymmv.
